src/Kodify/BlogBundle/Controller/CommentController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Kodify\BlogBundle\Controller;
+
+
+use Kodify\BlogBundle\Entity\Comment;
+use Kodify\BlogBundle\Form\Type\CommentType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class CommentController extends Controller
+{
+
+    public function createAction(Request $request)
+    {
+        $postRepository = $this->get('kodify.repository.post');
+        $postId = $request->get('id');
+        $post = $postRepository->find($postId);
+
+        if (!$post) {
+            throw new NotFoundHttpException('Post not found');
+        }
+
+        $comment = new Comment();
+        $comment->setPost($post);
+        $options = [
+            'action' => $this->get('router')->generate('comment_post', ['id' => $postId]),
+        ];
+        $form = $this->get('form.factory')->create(new CommentType(), $comment, $options);
+
+        $form->handleRequest($request);
+        if ($form->isValid()) {
+            $em = $this->get('doctrine')->getManager();
+            $em->persist($comment);
+            $em->flush();
+
+            $postId = $comment->getPost()->getId();
+
+            return $this->redirectToRoute('view_post', ['id' => $postId]);
+        }
```

Comment:
Do you mean, when it is valid? In this case the user should see his message in the target of the redirect, is true that we can show a flash message also to improve the feedback. When is invalid the user should see the errors in the form page.